phobos/io/representation.py

```diff
@@ -128,8 +128,10 @@ def __init__(self, *args, rgba=None, **kwargs):
         elif count == 0:
             self.rgba = None
         if self.rgba is not None:
-            if len(self.rgba) == 3:
+            if len(self.rgba) == 3 and type(args) is list:
                 self.rgba += [1.]
+            elif len(self.rgba) == 3 and type(args) is tuple:
+                self.rgba += (1.,)
```

Comment:
We should convert it to list here. We don't want to store a tuple

Reply:
Makes sense